### DISCUSSION TITLE
Alertmanager: Support watchdog alerts

### DISCUSSION THREAD

**Discussion by Poweranimal** [grafana/mimir#2083] [Help and support] [answered]
#### Is your feature request related to a problem? Please describe.

In order to know, if the Alertmanager's receivers are properly configured, a regular occurring alert (watchdog-alert) should send a notification to the configured receivers.

#### Describe the solution you'd like

It seems that as of now the Alertmanager doesn't fire watchdog alerts.

Though I'm not sure, if the Prometheus Alertmanager fires wachdog alerts or if it's Prometheus server.
If Prometheus server fires watchdog alerts, I wonder of one should then fire watchdog alerts by Mimir-ruler.
Reason: If one uses Mimir, it makes more sense to run Prometheus in agent mode. The agent mode of Prometheus doesn't support alerting. That said it would make sense to send the alerts by Mimir-ruler.

I look forward to your response.

**Comment by pracucci** [comment, marked as answer]
Have you considered to configure an always firing alert in the ruler, then the ruler sends the firing alert event to the Alertmanager, where you can configure the receivers. Would that solve your issue?

**Comment by Poweranimal** [comment]
Thank you very much for your reply.
No, I haven't considered using an always firing rule.
I've also noticed that such a rule is created by default by [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) [here](https://github.com/prometheus-community/helm-charts/blob/5ce650b75c3bb4cc85e0f7c23ac2614bb37d67de/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/general.rules.yaml#L42).

Thanks a lot for your help.
I'm going to create such a rule.
